src/wovnio/html/HtmlConverter.php

```diff
@@ -45,64 +44,39 @@ public function __construct($html, $encoding, $token, $store = null, $headers =
    *
    * @return array converted html and HtmlReplaceMarker
    */
-  public function convertToAppropriateForApiBody($removeParts = true)
+  public function insertSnippetAndHreflangTags()
   {
-    if ($this->encoding && in_array($this->encoding, self::$supported_encodings)) {
-      $encoding = $this->encoding;
-    } else {
-      // Encoding detection uses 30% of execution time for this method.
-      $encoding = mb_detect_encoding($this->html, self::$supported_encodings);
-    }
-
-    $dom = simple_html_dom::str_get_html($this->html, $encoding, false, false, $encoding, false);
-    $this->insertSnippet($dom);
-    if (isset($this->store) && isset($this->headers)) {
-      $this->insertHreflangTags($dom);
-    }
-
-    $marker = new HtmlReplaceMarker();
-    if ($removeParts) {
-      $this->removeWovnIgnore($dom, $marker);
-      $this->removeForm($dom, $marker);
-      $this->removeScript($dom, $marker);
-    }
-
-    $converted_html = $dom->save();
+    $this->html = $this->insertSnippet($this->html);
+    $this->html = $this->insertHreflangTags($this->html);
 
-    // Without clear(), Segmentation fault will be raised.
-    // @see https://sourceforge.net/p/simplehtmldom/bugs/103/
-    $dom->clear();
-    unset($dom);
-
-    if ($removeParts) {
-      $converted_html = $this->removeBackendWovnIgnoreComment($converted_html, $marker);
-    }
-    return array($converted_html, $marker);
+    return array($this->html);
   }
 
   /**
    * Insert wovn's snippet to ensure snippet is always inserted.
    * When snippet is always inserted, do nothing
    *
-   * @param simple_html_dom_node $dom
+   * @param simple_html_dom_node $html
```

Comment:
param is not simple_html_dom_node anymore I think, right?

Reply:
You're right. I will change. Thank you

Reply:
Done